README.md

```diff
@@ -176,32 +176,124 @@ Properties starting with a `$` are special properties. It must be one of the fol
+       <td>Yes</td>
+       <td>Yes</td>
+    </tr>
+    <tr>
+       <td>$range</td>
+       <td>Device → Controller</td>
+       <td>
+        Describes what are valid values for this property.
+       </td>
+       <td>
+             <ul>
+                <li><code>from:to</code> Describes a range of values e.g. <code>10:15</code>. <br>Valid for datatypes <code>integer</code>, <code>float</code> </li>
+                <li><code>value,value,value</code> for enumerating all valid values. Escape <code>,</code> by using <code>,,</code>. e.g. <code>A,B,C</code> or <code>ON,OFF,PAUSE</code>. <br>Valid for datatypes <code>enum</code> </li>
+                <li><code>/regex/</code> to provide a regex that can be used to match the value. e.g. <code>/[A-Z][0-9]+/g</code>. <br>Valid for datatype <code>string</code></li>
+            </ul>
+       </td>
+       <td>Yes</td>
+       <td>Yes</td>
+    </tr>
+    
+    
+</table>
+
+
 For example, our `686f6d6965` above would send:
 
 ```
 homie/686f6d6965/temperature/$type → temperature
 homie/686f6d6965/temperature/$properties → degrees,unit
-homie/686f6d6965/temperature/unit → c
+homie/686f6d6965/temperature/degrees/$settable → false
+homie/686f6d6965/temperature/degrees/$unit → C
+homie/686f6d6965/temperature/degrees/$datatype → float
+homie/686f6d6965/temperature/degrees/$range → -20.0:60
 homie/686f6d6965/temperature/degrees → 12.07
 
 homie/686f6d6965/humidity/$type → humidity
 homie/686f6d6965/humidity/$properties → percentage
+homie/686f6d6965/humidity/percentage/$settable → false
+homie/686f6d6965/humidity/percentage/$unit → %
+homie/686f6d6965/humidity/percentage/$datatype → integer
+homie/686f6d6965/humidity/percentage/$range → 0:100
 homie/686f6d6965/humidity/percentage → 79
 ```
 
 A LED strip would look like this. Note that the topic for a range properties is the name of the property followed by a `_` and the index getting updated:
 
 ```
 homie/ledstrip-device/ledstrip/$type → ledstrip
-homie/ledstrip-device/ledstrip/$properties → led[1-3]:settable
+homie/ledstrip-device/ledstrip/$properties → led[1-3]
+
+homie/ledstrip-device/ledstrip/led_1/$settable → true
+homie/ledstrip-device/ledstrip/led_1/$unit → LED Status
+homie/ledstrip-device/ledstrip/led_1/$datatype → enum
+homie/ledstrip-device/ledstrip/led_1/$range → on,off
 homie/ledstrip-device/ledstrip/led_1 → on
+
+homie/ledstrip-device/ledstrip/led_2/$settable → true
+homie/ledstrip-device/ledstrip/led_2/$unit → LED Status
+homie/ledstrip-device/ledstrip/led_2/$datatype → enum
+homie/ledstrip-device/ledstrip/led_2/$range → on,off
 homie/ledstrip-device/ledstrip/led_2 → off
+
+homie/ledstrip-device/ledstrip/led_3/$settable → true
+homie/ledstrip-device/ledstrip/led_3/$unit → LED Status
+homie/ledstrip-device/ledstrip/led_3/$datatype → enum
+homie/ledstrip-device/ledstrip/led_3/$range → on,off
 homie/ledstrip-device/ledstrip/led_3 → on
 ```
 
```

Comment:
The following (unchanged) section is inconsistent with the LED example.  I like the new definition with "On" or "Off" as state, so I propose to update the following section to make use of it. (No more "On"-state that is set to true or false).